xds/internal/xdsclient/xdsresource/cluster_resource_type.go

```diff
@@ -103,15 +103,11 @@ func (l *ClusterResourceData) Raw() *anypb.Any {
 	return l.Resource.Raw
 }
 
-// ClusterResourceWatcher wraps the callbacks invoked by the xDS client
-// implementation for different events corresponding to the listener resource
-// being watched.
+// ClusterResourceWatcher wraps the callbacks to be invoked for different events
+// corresponding to the cluster resource being watched.
 type ClusterResourceWatcher interface {
```

Comment:
How do you feel about `ClusterWatcher` (etc)?

Reply:
Done.